source/features/blocks/components/BlockCreationRow.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
```

Comment:
@DeeJayElly Same here. Would you please use functional component instead which is better in terms of performance?

Reply:
Fixed